src/constants.ts

```diff
@@ -14,4 +15,26 @@ export const defaultGameInfo: GameInfo = {
   isQuestionPointsHidden: false,
 };
 
-export const MIN_NUM_OF_CATEGORIES = 2;
+export const tilesBGColors = [
+  '#f5866c',
+  '#e756cc',
+  '#4b8ef6',
+  '#bea798',
+  '#00bfe1',
+  '#a5db00',
+  '#fbd103',
+  '#7ecfcf',
+  '#95a5b5',
+  '#47ff76',
+  '#fb8065',
+  '#5151fa',
+  '#9c4afb',
```

Comment:
Instead of so many colors we should have 2 or 3 only of different shades. Otherwise it will look like rainbow or kids app.

Reply:
?